src/components/auth/auth.dto.ts

```diff
@@ -34,3 +34,12 @@ export class LoginOutput {
 
   // TODO Global Permissions
 }
+
+@InputType()
+export abstract class ResetPasswordInput {
+  @Field()
+  token: string;
+
+  @Field()
+  password: string;
```

Comment:
```suggestion
  @Field()
  readonly token: string;

  @Field()
  readonly password: string;
```

This ensure data is correctly processed and silly things are not done with passing `input` around; it is also consistent with all other inputs.